module/VuFind/src/VuFind/AjaxHandler/GetItemStatuses.php

```diff
@@ -127,7 +130,8 @@ protected function translateList($transPrefix, $list)
     {
         $transList = [];
         foreach ($list as $current) {
-            $transList[] = $this->translateWithPrefix($transPrefix, $current);
+            // $current can be an array if pickValue() is called with callnumbers and a translation prefix
```

Comment:
I'm sorry if this caused confusion. Maybe I just don't know how to word it correctly.

`translateList` will not be called for callnumbers without translation prefixes since `pickValue()` would just return the list:
`return $transPrefix ? $this->translateList($transPrefix, $list) : $list;`

Reply:
Okay, my bad. I didn't notice that. I believe you could still remove the " and a translation prefix" part. But since this is not wrong, I don't mind leaving this as is.